clients/ui/bff/internal/helpers/logging.go

```diff
@@ -0,0 +1,113 @@
+package helper
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/kubeflow/model-registry/ui/bff/internal/constants"
+	"io"
+	"log/slog"
+	"net/http"
+	"slices"
+)
+
+func GetContextLoggerFromReq(r *http.Request) *slog.Logger {
+	return GetContextLogger(r.Context())
+}
+
+func GetContextLogger(ctx context.Context) *slog.Logger {
+	logger := ctx.Value(constants.TraceLoggerKey).(*slog.Logger)
```

Comment:
Shall we use logger, ok := ctx.Value(constants.TraceLoggerKey).(*slog.Logger) ?

Reply:
The nil check below should handle any error cases, that said I should at least use `logger, _ := ...` to explicitly indicate I'm ignoring the error.

Reply:
Actually I can use the "ok" variable instead of the nil check.